randovania/games/dread/logic_database/Ferenia.txt

```diff
@@ -661,8 +661,10 @@ Extra - asset_id: collision_camera_012
               All of the following:
                   Lay Bomb
                   Any of the following:
-                      Space Jump or Movement (Intermediate)
+                      Space Jump
                       Movement (Beginner) and Use Spin Boost
+                      # https://www.youtube.com/watch?v=AaSlxp8GajA
+                      Movement (Advanced)
```

Comment:
While we're in here, how does one get to the item using just Power Bomb and Movement?

Reply:
The same process as the video, get above the blocks before they reform.